pandunia-lekse.csv

```diff
@@ -2919,6 +2973,7 @@ tanure|||fas:heb:tanur, ara:(tannūr), eng:tandoor, athanor, tur:tandır, hin:
 tanzania|dex|TZ||Tanzania||Tanzania|||||||||||||Tanzanio|Tansania|Tanzania
 tapa||||plug (stop with a plug)||taponar||||||||||||||korkittaa (tukkia)|zatkać, ztykać
 tape|||spa:tur:tapa, por:tampa, ell:τάπα (tapa), bul:тапа (tapa), eng:tap|stopper (cap, peg, tap)||tapón (tapa)||||||||||||||tulppa (tappi, korkki)|korek, zatyczka, szpunt
+taraxkone|||ara:طرخشقون (ṭaraḵšaqūn), por:taráxaco|dandelion|dandélion|dientes de león (panaderos)|dente-de-leão (taráxaco)|одуванчик||蒲公英|タンポポ||||||||||
```

Comment:
I wouldn't burden my memory with a long root like that because of just one flower. How about "singodantofule"? In Finnish we call it "makanfule". ;)

Reply:
I like "makanfule". It's way shorter than "singodantofule".

Reply:
OK. :) Actually, also "singofule" could make sense because the flower kind of looks like a lion but I don't know are there other "lion flowers" already in existence.